### ISSUE TITLE
WebGl support on windows?

### ISSUE THREAD

**Issue by 2youyouo2** [adobe/brackets#10880]
hey guys!
I was trying to embed a webgl content in the brackets, for game editting.
It's ok on Mac OSX, but failed on Windows.
I have tried to add --enable-webgl --ignore-gpu-blacklist to start command, but still can't get a webgl context on Windows in brackets.
I have tested webgl with http://threejs.org/examples/#webgl_animation_cloth in chrome and node-webkit, it's ok.
So i think my windows supports webgl.

Is there a way to make it right?
Thanks for readding!

**Comment by abose** [comment]
The CEF version integrated with brackets should have support for WebGl.
But it may be not initialized or enabled or worse, the DLLs may not be part of brackets.
Angle and Direct3D support
    d3dcompiler_43.dll (required for Windows XP)
    d3dcompiler_46.dll (required for Windows Vista and newer)
    libEGL.dll
    libGLESv2.dll
  Note: Without these components HTML5 accelerated content like 2D canvas, 3D
  CSS and WebGL will not function.

**Comment by abose** [comment]
You could build brackets shell from source 
https://github.com/adobe/brackets-shell/wiki/Building-brackets-shell
The built version has all the D3D and webgl DLL's.

**Comment by 2youyouo2** [comment]
I rebuild brackets for windows, and it works well ! ^_^
Thanks for your answer, @abose.

**Comment by abose** [comment]
:+1: Moving to backlog, confirm if we should have integrated the DLL's in windows natively as mac/linux builds already ships with the bins.

**Comment by abose** [comment]
@2youyouo2 If you need brackets to integrate the DLL's in the windows end for writing extensions feel free to reopen this issue.

**Comment by 2youyouo2** [comment]
It's important for me, but i don't know anyone else need  webgl in brackets.
So i reopen this issue to see whether anyone else need it.

**Comment by cubuspl42** [comment]
Yes, please! I vote for WebGL in bracket-shell by default! :+1:

**Comment by abose** [comment]
Are you able to run webgl content on mac builds?

**Comment by cubuspl42** [comment]
Yes, in official Bracket, out out the box.

**Comment by abose** [comment]
Including the DLL's would add around 4 MB to the installer footprint. 
@zaggino @nethip @MarcelGerber @petetnt

**Comment by petetnt** [comment]
If there are no technical reasons why WebGL support absolutely shouldn't be added :+1: from me. 

It opens the world of possibilities for creative stuff in the future and it seems to be already possible on Mac builds so I am willing to take the 4 MB footprint hit.

**Comment by zaggino** [comment]
Yeah, I also don't think 4 MB is anything to be worried about. If is as simple as adding a DLL to the build, lets do it.

**Comment by Denisov21** [comment]
Yes, please! I vote for WebGL in Bracket! :+1:
